flow_control/flow.py

```diff
@@ -128,6 +129,22 @@ class FlowCheckPointXblock(StudioEditableXBlockMixin, XBlock):
                        'target_id',
                        'message')
 
+    def validate_field_data(self, validation, data):
+        """
+        Validate this block's field data
+        """
+
+        if data.tab_to <= 0:
```

Comment:
this could go in a different PR, with some other fixes from the feedback we have got